src/network/network_server.cpp

```diff
@@ -202,16 +202,14 @@ struct PacketWriter : SaveFilter {
  * Create a new socket for the server side of the game connection.
  * @param s The socket to connect with.
  */
-ServerNetworkGameSocketHandler::ServerNetworkGameSocketHandler(SOCKET s) : NetworkGameSocketHandler(s)
+ServerNetworkGameSocketHandler::ServerNetworkGameSocketHandler(SOCKET s, NetworkAddress client_address) : NetworkGameSocketHandler(s, _network_client_id++), status(STATUS_INACTIVE), receive_limit(_settings_client.network.bytes_per_frame_burst), client_address(client_address)
 {
-	this->status = STATUS_INACTIVE;
-	this->client_id = _network_client_id++;
-	this->receive_limit = _settings_client.network.bytes_per_frame_burst;
-
 	/* The Socket and Info pools need to be the same in size. After all,
 	 * each Socket will be associated with at most one Info object. As
 	 * such if the Socket was allocated the Info object can as well. */
 	static_assert(NetworkClientSocketPool::MAX_SIZE == NetworkClientInfoPool::MAX_SIZE);
+
+	DEBUG(net, 1, "[%s] Accepting client #%d connection from %s", ServerNetworkGameSocketHandler::GetName(), this->client_id, this->client_address.GetHostname());
```

Comment:
Guess you might as well use `GetClientIP()` everywhere instead of `this->client_address.GetHostname()` :) (not really sure why it needs a wrapper honestly, but as you already use it a few times, why not everywhere :D )

Reply:
I originally started by using `client_address.GetHostName()`, and then I discovered `GetClientIP()`...

You are right, the change needs to be homogeneous: I will try and see which one is better.